src/types/Card.ts

```diff
@@ -0,0 +1,7 @@
+export interface Card {
```

Comment:
you describe a movie
```suggestion
export interface Movie {
```